bin/pulsar-managed-ledger-admin

```diff
@@ -20,11 +20,19 @@
 
 import argparse
 import traceback
-from google.protobuf.text_format import Merge
-from google.protobuf.text_format import MessageToString
 import sys
-    
-from proto import MLDataFormats_pb2
+try:
+    from google.protobuf.text_format import Merge
+    from google.protobuf.text_format import MessageToString
+except Exception as missingLib:
+    sys.exit("You need python protobuf library. See: https://github.com/google/protobuf/tree/master/python")
```

Comment:
yes, that would also work..I kept url if anyone wants to install specific tag/version. But, I think keeping `pip install protobuf` will look more simpler. will change it.